code/modules/atmospherics/machinery/pipes/bluespace_pipe.dm

```diff
@@ -0,0 +1,50 @@
+GLOBAL_LIST_EMPTY(bluespace_pipes)
+
+/obj/machinery/atmospherics/pipe/bluespace
+	name = "bluespace pipe"
+	desc = "Transmits gas across large distances of space. Developed using bluespace technology."
+	icon = 'icons/obj/atmospherics/pipes/bluespace.dmi'
+	icon_state = "map"
+	dir = SOUTH
+	initialize_directions = SOUTH
+	device_type = UNARY
+	can_buckle = FALSE
+
+/obj/machinery/atmospherics/pipe/bluespace/New()
+	icon_state = "pipe"
+	GLOB.bluespace_pipes += src
+	..()
```

Comment:
Initialize and . = ..() first

Reply:
I'll open another PR in a few minutes to change all atmos machines to use initialize.

Reply:
Too many pipenet related things broke during mapload when I tried changing to `Initialize` so I'm just gonna leave it for now if that's ok with you.